The job **`mingw-check`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/11765700997/job/32772336321) [(plain)](https://github.com/rust-lang/rust/commit/e1d09a5e8332e85253d57836ee3f4e5dbfd5697a/checks/32772336321/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
#16 3.052 Building wheels for collected packages: reuse
#16 3.053   Building wheel for reuse (pyproject.toml): started
#16 3.296   Building wheel for reuse (pyproject.toml): finished with status 'done'
#16 3.297   Created wheel for reuse: filename=reuse-4.0.3-cp310-cp310-manylinux_2_35_x86_64.whl size=132720 sha256=026f3bb0f1aa8090b861fd0a0939cb1a782396d84c8aab7875096557d637a0f6
#16 3.297   Stored in directory: /tmp/pip-ephem-wheel-cache-04_y9oe3/wheels/3d/8d/0a/e0fc6aba4494b28a967ab5eaf951c121d9c677958714e34532
#16 3.300 Installing collected packages: boolean-py, binaryornot, tomlkit, reuse, python-debian, markupsafe, license-expression, jinja2, chardet, attrs
#16 3.695 Successfully installed attrs-23.2.0 binaryornot-0.4.4 boolean-py-4.0 chardet-5.2.0 jinja2-3.1.4 license-expression-30.3.0 markupsafe-2.1.5 python-debian-0.1.49 reuse-4.0.3 tomlkit-0.13.0
#16 3.696 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#16 DONE 3.8s
---
##[endgroup]
[CI_JOB_NAME=mingw-check]
[CI_JOB_NAME=mingw-check]
debug: `DISABLE_CI_RUSTC_IF_INCOMPATIBLE` configured.
/checkout/src/ci/run.sh: line 184: syntax error near unexpected token `fi'
##[error]Process completed with exit code 2.
```

</details>

--- a/src/ci/run.sh
+++ b/src/ci/run.sh
@@ -179,7 +179,9 @@ else
   fi
 
   if [ "$NO_DOWNLOAD_CI_RUSTC" = "" ]; then
-    RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --set rust.download-rustc=if-unchanged"
+    # disabled for now, see https://github.com/rust-lang/rust/issues/131658
+    #RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --set rust.download-rustc=if-unchanged"
+    true
   fi
 fi
 

--- a/tests/run-make/version-verbose-commit-hash/rmake.rs
+++ b/tests/run-make/version-verbose-commit-hash/rmake.rs
@@ -3,9 +3,6 @@
 // test ensures it will not be broken again.
 // See https://github.com/rust-lang/rust/issues/107094
 
-// FIXME(#132845): temporarily disabled to get revert through
-//@ ignore-test
-
 //@ needs-git-hash
 
 use run_make_support::{bare_rustc, bare_rustdoc, regex};

